scripts/sync_s3_env.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+# Syncs two environments S3 buckets and their config files.
+# Can only go from prod -> dev <-> dev-green
+
+set -eu
+
+SCREENS_BUCKET="s3://mbta-screens/screens"
+SCREENS_JSON="s3://mbta-ctd-config/screens/screens"
+PENDING_SCREENS_JSON="s3://mbta-ctd-config/screens/pending-screens"
+
+DELETE_FLAG=false
+SOURCE_ENV=""
+DEST_ENV=""
+
+while getopts "Dhs:d:" opt; do
+  case $opt in
+  D)
+    echo "Running with delete flag - will delete files in destination environment that do not exist in source enviornment"
+    ;;
+  s)
+    SOURCE_ENV=$OPTARG
+    ;;
+  d)
+    DEST_ENV=$OPTARG
+    ;;
+  h)
+    echo "Usage: $0 [-d] -f <screens-env-source-name> -t <screens-env-dest-name>" >&2
+    exit 0
+    ;;
+  *)
+    echo "Usage: $0 [-d] -f <screens-env-source-name> -t <screens-env-dest-name>" >&2
+    exit 1
+    ;;
+  esac
+done
+
+case $SOURCE_ENV in
+prod | dev | dev-green) true ;;
+*)
+  echo "Usage: $0 [-D] -s <screens-env-source-name> -d <screens-env-dest-name>" >&2
+  echo "<screens-env-source-name> should be one of the following: prod | dev | dev-green" >&2
+  exit 1
+  ;;
+esac
+
+case $DEST_ENV in
+dev | dev-green) true ;;
+*)
+  echo "Usage: $0 [-d] -s <screens-env-source-name> -d <screens-env-dest-name>" >&2
+  echo "<screens-env-dest-name> should be one of the following: dev | dev-green" >&2
+  exit 1
+  ;;
+esac
+
+SOURCE_BUCKET="$SCREENS_BUCKET-$SOURCE_ENV"
+DEST_BUCKET="$SCREENS_BUCKET-$DEST_ENV"
+
+SCREENS_SOURCE_JSON="$SCREENS_JSON-$SOURCE_ENV.json"
+SCREENS_DEST_JSON="$SCREENS_JSON-$DEST_ENV.json"
+
+PENDING_SCREENS_SOURCE_JSON="$PENDING_SCREENS_JSON-$SOURCE_ENV.json"
+PENDING_SCREENS_DEST_JSON="$PENDING_SCREENS_JSON-$DEST_ENV.json"
+
+printf '%s' "Copy *$SOURCE_ENV* screens assets and configs into *$DEST_ENV*? "
+read -r answer
+case $answer in
+[Yy]*)
+  if [ "$DELETE_FLAG" = true ]; then
+    aws s3 sync "$SOURCE_BUCKET" "$DEST_BUCKET" --exact-timestamps --delete --dryrun
```

Comment:
These all have the `--dryrun` flag, so will not do anything for real. Left over from testing?